### PR TITLE
Always deploy same revision to group

### PR DESCRIPTION
**Pull request by gigxz**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

This is an untracked change, just hacked on it a little because I accidentally deployed the wrong commits because of the current behavior.

**Current behavior**: when deploying a group, each deployment will deploy the commit/branch that you're currently on at the time when it reaches that group
**Proposed behavior**: when deploying a group, all deployments will use the commit/branch that you were on when you kicked off the group

**How I tested**: by commenting out `roll_out.run!` and `_add_latest_tag!` in the `run!` script

## Type of change
- [x] New feature (adds functionality)

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: config/deploy/docker/lib/deployer.rb [hunk 1]
```diff
@@ -261,6 +263,8 @@ def _pre_cache_version
 
   def _set_image_tag!
     if ENV['IMAGE_TAG']
+      # TODO this overrides the specified revision/version - when is this used?
+      # seem like this would've already caused a mismatch with _set_revision
       self.image_tag = ENV['IMAGE_TAG']
       self.image_tag_latest = 'latest-' + ENV['IMAGE_TAG']
```
Comment (gigxz) [inline config/deploy/docker/lib/deployer.rb]: open question here

Comment (yakloinsteak) [reply config/deploy/docker/lib/deployer.rb]: It's just for debugging and not part of normal deployments.

**Review by yakloinsteak — APPROVED** [review]
Looks reasonable to me. I've hit this problem once or twice myself.

**Review by eanders — APPROVED** [review]
This looks great, thanks @gigxz !

File: config/deploy/docker/lib/deployer.rb (hunk 1 above)
Comment (gigxz) [reply config/deploy/docker/lib/deployer.rb]: okay great, I updated the comment to say that

**Comment by gigxz** [comment]
Build is failing due to unrelated vulnerability issue, merging

**Pull request merged**